api/src/integration/java/org/pmiops/workbench/google/DirectoryServiceImplIntegrationTest.java

```diff
@@ -25,6 +25,14 @@ public void testDirectoryServiceUsernameIsTaken() {
     assertThat(service.isUsernameTaken("directory-service")).isTrue();
   }
 
+  @Test
+  public void testCreateAndDeleteTestUser() {
+    service.createUser("Integration", "Test", "integration.test", "notasecret");
```

Comment:
So... this actually creates a real Gsuite user and deletes it.... are there any ramifications for this in terms of cost?

Reply:
On another note: if our test dies / loses connectivity after creating a user but before deleting it, this test will start failing. 

You might want to issue a deleteUser first before doing the create just to handle that case.

Reply:
That would work, but I expect us to run these manually and rarely right now, so I'm okay with leaving in the fragility.